python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -420,11 +420,85 @@ def __keyPress( column, pathListing, event ) :
 
 	return False
 
+def __dragEnter( path, pathListing, event ) :
+
+	column = pathListing.columnAt( event.line.p0 )
+	if path is None or not isinstance( column, GafferSceneUI.Private.InspectorColumn ) :
+		return False
+
+	inspectionContext = path.inspectionContext()
+	if inspectionContext is None :
+		return False
+
+	return True
+
+def __dragLeave( path, pathListing, event ) :
+
+	return True
+
+def __dragMove( path, pathListing, event ) :
+
+	column = pathListing.columnAt( event.line.p0 )
+	if path is None or not isinstance( column, GafferSceneUI.Private.InspectorColumn ) :
+		return False
+
+	inspectionContext = path.inspectionContext()
+	if inspectionContext is None :
+		return False
+
+	return True
```

Comment:
Do we actually need to implement `dragMove`? Once we've accepted a drag for a cell, I don't think we care about the precise position do we?

I wasn't going to bring this up, but I also wonder if `event.line` should be relative to the cell, since that's the granularity at which we do enter/leave. I don't think it matters right now though. I almost wonder if we should drop `dragMove()` for simplicity, and because I don't think there's anything we could use it for right now.

Reply:
Scratch all that - I see we do use `move` in a later commit, so we can react to keypresses.